x/stake/client/rest/query.go

```diff
@@ -492,6 +492,7 @@ func validatorsHandlerFn(cliCtx context.CLIContext, cdc *wire.Codec) http.Handle
 			return
 		}
 
+		w.Header().Set("Content-Type", "application/json")
```

Comment:
is this the only content change? everything else looks like additional coverage (which is great)

Reply:
Yes.